src/utils/utils.ts

```diff
@@ -84,7 +83,7 @@ export function setDeepValue(obj: any, path: any, value: any) {
 export function shorten(str: string, maxLen: number, separator = ' ') {
   if (!str) return;
   if (str.length <= maxLen) return str;
-  return str.substr(0, str.lastIndexOf(separator, maxLen)) + '..';
+  return str.substring(0, str.lastIndexOf(separator, maxLen)) + '..';
 }
 
 export function truncateString(str: string, num: number) {
```

Comment:
Not specific to this PR, but wonder if this could be replaced with https://www.patternfly.org/v4/components/truncate

Reply:
Yes, I agree, remove this function and use patternfly instead :ok_hand: